### ISSUE TITLE
Enhance a11y for `Flip`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5405]
name: Flip

### Description
8 accessibility violations were found in the `Flip`.

#### [Components / Flip / Basic](https://yamada-ui.github.io/yamada-ui/?path=/story/components-flip--basic)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Flip / Orientation](https://yamada-ui.github.io/yamada-ui/?path=/story/components-flip--orientation)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Flip / Duration](https://yamada-ui.github.io/yamada-ui/?path=/story/components-flip--duration)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Flip / Delay](https://yamada-ui.github.io/yamada-ui/?path=/story/components-flip--delay)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Flip / Transition](https://yamada-ui.github.io/yamada-ui/?path=/story/components-flip--transition)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Flip / Disabled](https://yamada-ui.github.io/yamada-ui/?path=/story/components-flip--disabled)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Flip / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-flip--read-only)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Flip / CustomControl](https://yamada-ui.github.io/yamada-ui/?path=/story/components-flip--custom-control)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
